assets/src/features/room-page/components/DisabledTrackIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import MicrophoneOff from "../icons/MicrophoneOff";
+import { TrackWithId } from "../../../pages/types";
+
+type DisabledMicIconProps = {
+  isLoading: boolean;
+};
+
+export const DisabledMicIcon = ({ isLoading }: DisabledMicIconProps) => {
+  return (
+    <div className="flex h-8 w-8 flex-wrap content-center justify-center rounded-full bg-white">
+      <MicrophoneOff className={isLoading ? "animate-spin" : ""} fill="#001A72" />
```

Comment:
Please use the `brand-dark-blue-500` class defined in tailwind config for `#001A72` value